smi-conformance/vendor/github.com/Azure/go-autorest/autorest/adal/persist.go

```diff
@@ -0,0 +1,73 @@
+package adal
```

Comment:
Is it wise to add all the dependency boilerplate logic in the repository itself rather than fetching the dependencies through go mod

Reply:
We do want to do package management through Go modules.

Reply:
@kushthedude  @leecalcote  go mod has a bug  which would fail when fetching from a private repository if present. To avoid such instances, I skipped the go mod management. This package would work without any issues but we wont be able to use private packages.

Reply:
No issues @kumarabd , we can skip this for now!